tests/test_publish_dafsa.py

```diff
@@ -0,0 +1,148 @@
+import os
+import tempfile
+import unittest
+from unittest import mock
+
+import requests
+import responses
+from kinto_http import Client, KintoException
+
+
+from commands.publish_dafsa import (
+    get_latest_hash,
+    download_resources,
+    prepare_dafsa,
+    remote_settings_publish,
+    publish_dafsa,
+    PREPARE_TLDS_PY,
+    MAKE_DAFSA_PY,
+    LIST_URL,
+    BUCKET_ID,
+    COLLECTION_ID,
+    RECORD_ID,
+    COMMIT_HASH_URL,
+)
+
+
+class TestUtilMethods(unittest.TestCase):
+    def test_get_latest_hash(self):
+        self.assertEqual(len(get_latest_hash(COMMIT_HASH_URL)), 40)
+        with self.assertRaises(IndexError):
+            get_latest_hash(COMMIT_HASH_URL + "c")
+        with self.assertRaises(requests.exceptions.HTTPError):
+            get_latest_hash("".join(COMMIT_HASH_URL.split("repo")))
+
+    def test_download_resources(self):
+        with tempfile.TemporaryDirectory() as tmp:
+            download_resources(tmp, PREPARE_TLDS_PY, MAKE_DAFSA_PY, LIST_URL)
+            self.assertEqual(
+                sorted(os.listdir(tmp)),
+                sorted(["public_suffix_list.dat", "prepare_tlds.py", "make_dafsa.py"]),
+            )
+            with self.assertRaises(requests.exceptions.HTTPError):
+                download_resources(tmp, PREPARE_TLDS_PY + "c"),
```

Comment:
Shouldn't you pass other parameters here?

Reply:
This test is only to check if HTTPError is raised or not, that can be done with just one parameter.